### ISSUE TITLE
`yarn fetch` is conflating 2 images

### ISSUE THREAD

**Issue by dankohn** [cncf/landscape#512]
These two entries are point to different images:
```yml
          - item:
            name: Codefresh
            homepage_url: 'https://codefresh.io/'
            logo: ./hosted_logos/codefresh.svg
            twitter: 'https://twitter.com/codefresh'
            crunchbase: 'https://www.crunchbase.com/organization/code-fresh'
          - item:
            name: Codefresh Registry
            homepage_url: 'https://codefresh.io/docs/docs/docker-registries/codefresh-registry/'
            logo: 'https://codefresh.io/wp-content/uploads/2018/03/Codefresh-Registry.svg'
            crunchbase: 'https://www.crunchbase.com/organization/code-fresh'
```
But `yarn fetch` is having both of them served by https://landscape.cncf.io/logos/codefresh-registry.svg.

**Comment by ZeusTheTrueGod** [comment]
![image](https://user-images.githubusercontent.com/230982/38571511-616c2ed4-3cf9-11e8-94dc-d094e3dbfada.png)

**Comment by ZeusTheTrueGod** [comment]
![image](https://user-images.githubusercontent.com/230982/38571544-74c00582-3cf9-11e8-8e92-395e5dc15075.png)

**Comment by ZeusTheTrueGod** [comment]
I see the problem now, at first glance I have not noticed the word 'registry'

**Comment by ZeusTheTrueGod** [comment]
That is a part of #504

**Comment by ZeusTheTrueGod** [comment]
Will fix in #504
